src/device/device.ts

```diff
@@ -431,7 +431,16 @@ export class MicrobitWebUSBConnection extends EventEmitter {
   serialWrite(data: string): Promise<void> {
     return this.withEnrichedErrors(async () => {
       if (this.connection) {
-        return this.connection.daplink.serialWrite(data);
+        // Using WebUSB/DAPJs we're limited to 64 byte packet size with a two byte header.
+        // https://github.com/microbit-foundation/python-editor-next/issues/215
```

Comment:
Should this be in DAPJs or at least documented on serialWrite?

Reply:
Chatted to @microbit-carlos and did a bit more digging.

I think if we were using the node-usb DAPJs transport then this might have been addressed automatically (by libusb). Not had a chance to try yet. See docs at https://github.com/tessel/node-usb#outendpoint

So I think it's worth following up and seeing if the fix should be in the WebUSB transport. Not going to hold this up on that though, but I'll reopen the issue for follow up.